old-src/endpoints/v1.es6.js

```diff
@@ -1070,6 +1070,10 @@ class APIv1Endpoint {
           // user information.
           const data = {
             name: '',
+            loid: body.loid,
+            loid_created: typeof body.loid_created === 'number'
```

Comment:
right now, we get back a time in ms. this seems incompatible with our current `loidcreated` cookies, and throws errors when we try to use them. i'll make a ticket to get this changed, but for now, this is a safety check.